tests/formats/mmtf.cpp

```diff
@@ -250,4 +250,67 @@ TEST_CASE("Write files in MMTF format") {
         CHECK(topology.are_linked(topology.residue(0), topology.residue(1)));
         CHECK(!topology.are_linked(topology.residue(0), topology.residue(2)));
     }
+
+    SECTION("Structure not from RCSB") {
+        auto tmpfile = NamedTempPath(".mmtf");
```

Comment:
It would be interesting to see what an application like PyMol does with this file (it uses the same MMTF backend). My guess is that it wouldn't open, but I don't have the time to check.

Reply:
I converted the water.xyz test file to mmtf with this branch, and PyMol seems to load it fine.

Reply:
Great!